src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,11 @@
+html {
```

Comment:
The _normalize.scss file should contain CSS reset or normalize rules. Consider moving the custom font declaration to a separate file or the main SCSS file.